pkg/handler/repositories.go

```diff
@@ -9,14 +9,24 @@ import (
 	"github.com/content-services/content-sources-backend/pkg/dao"
 	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/content-services/content-sources-backend/pkg/models"
+	"github.com/davecgh/go-spew/spew"
 	"github.com/labstack/echo/v4"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 )
 
-func RegisterRepositoryRoutes(engine *echo.Group) {
+type RepositoryHandler struct {
+	RepositoryDao dao.RepositoryDao
+}
```

Comment:
so now the handler is an object, which lets you instantiate the handler with our mocked Dao and run requests against it.